cmd/argocd/commands/app.go

```diff
@@ -1746,6 +1768,24 @@ func NewApplicationSyncCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 	return command
 }
 
+func getAppNamesBySelector(ctx context.Context, appIf applicationpkg.ApplicationServiceClient, selector string) ([]string, error) {
+	appNames := []string{}
+	if selector != "" {
+		list, err := appIf.List(ctx, &applicationpkg.ApplicationQuery{Selector: pointer.String(selector)})
+		if err != nil {
+			return []string{}, nil
```

Comment:
Sorry to ask for one more change, but can we return this error instead of ignoring it? If there were, for example, a network issue, I wouldn't expect the command to fail silently.

Reply:
Sure thing. Think this might have been part of original code so I left alone, but I agree.